internal/cmd/cloud-signup/command.go

```diff
@@ -1,213 +1,36 @@
 package cloudsignup
 
 import (
-	"context"
-	"fmt"
-	"os"
-	"strings"
-
-	ccloudv1 "github.com/confluentinc/ccloud-sdk-go-v1-public"
-	"github.com/gogo/protobuf/types"
+	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 
-	"github.com/confluentinc/cli/internal/cmd/admin"
-	pauth "github.com/confluentinc/cli/internal/pkg/auth"
-	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
-	"github.com/confluentinc/cli/internal/pkg/countrycode"
 	"github.com/confluentinc/cli/internal/pkg/errors"
 	"github.com/confluentinc/cli/internal/pkg/form"
-	"github.com/confluentinc/cli/internal/pkg/log"
 	"github.com/confluentinc/cli/internal/pkg/utils"
 )
 
-type command struct {
-	*pcmd.CLICommand
-	userAgent     string
-	clientFactory pauth.CCloudClientFactory
-	isTest        bool
-}
-
-func New(prerunner pcmd.PreRunner, userAgent string, ccloudClientFactory pauth.CCloudClientFactory, isTest bool) *cobra.Command {
+func New() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "cloud-signup",
 		Short: "Sign up for Confluent Cloud.",
 		Args:  cobra.NoArgs,
 	}
-
-	c := &command{
-		CLICommand:    pcmd.NewAnonymousCLICommand(cmd, prerunner),
-		userAgent:     userAgent,
-		clientFactory: ccloudClientFactory,
-		isTest:        isTest,
-	}
-	cmd.RunE = c.cloudSignupRunE
-
-	cmd.Flags().String("url", "https://confluent.cloud", "Confluent Cloud service URL.")
+	cmd.RunE = cloudSignup
 
 	return cmd
 }
 
-func (c *command) cloudSignupRunE(cmd *cobra.Command, _ []string) error {
-	url, err := cmd.Flags().GetString("url")
+func cloudSignup(cmd *cobra.Command, _ []string) error {
+	utils.Println(cmd, "You will now be redirected to the Confluent Cloud sign up page in your browser.")
```

Comment:
How about adding the link to this text, "If you are not redirected, use the following link: https://..."